src/plugins/cordova-plugin-network-information/app-host-clobbers.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This clobbers are needed in order to be able to load the script `src/windows/NetworkInfoProxy.js` when running the simulation for `windows` platform.